site/content/tutorial/07-stores/02-auto-subscriptions/text.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Auto-subscriptions
+---
+
+The app in the previous example works, but there's a subtle bug — the `unsubscribe` function never gets called. If the component was instantiated and destroyed many times, this would result in a *memory leak*.
+
+One way to fix it would be to use the `onDestroy` [lifecycle hook](tutorial/ondestroy):
+
+```html
+<script>
+	import { onDestroy } from 'svelte';
+	import { count } from './stores.js';
+	import Incrementer from './Incrementer.svelte';
+	import Decrementer from './Decrementer.svelte';
+	import Resetter from './Resetter.svelte';
+
+	let count_value;
+
+	const unsubscribe = count.subscribe(value => {
+		count_value = value;
+	});
+
+	onDestroy(unsubscribe);
+</script>
+
+<h1>The count is {count_value}</h1>
+```
+
+It starts to get a bit boilerplatey though, especially if your component subscribes to multiple stores. Instead, Svelte has a trick up its sleeve — you can reference a store value by prefixing the store name with `$`:
+
+```html
+<script>
+	import { count } from './stores.js';
+	import Incrementer from './Incrementer.svelte';
+	import Decrementer from './Decrementer.svelte';
+	import Resetter from './Resetter.svelte';
+</script>
+
+<h1>The count is {$count}</h1>
+```
+
+You're not limited to using `$count` inside the markup, either — you can use it anywhere in the `<script>` as well, such as in event handlers or reactive declarations.
+
+> Any name beginning with `$` is assumed to refer to a store value. It's effectively a reserved character — don't declare your own variables with a `$` prefix.
```

Comment:
It's a minor thing, but I wonder whether it would be better to reword this to indicate that the compiler won't let you do this. It's less to worry about if you know that the compiler will disallow something and you won't just end up with a buggy component.